addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -340,9 +340,9 @@ def _compute_preview_move_data(self):
     def do_action(self):
         move_vals = json.loads(self.move_data)
         if self.action == 'change_period':
```

Comment:
```suggestion
    def do_action(self):
        move_vals = json.loads(self.move_data)
        self = self.with_context(skip_computed_taxes=True)
        if self.action == 'change_period':
```
I was even thinking something like this (to avoid redundancy) :wink:

Reply:
Scraping the characters off, right? 😄 I updated it